cache/lru_cache.cc

```diff
@@ -753,6 +754,8 @@ void LRUCache::WaitAll(std::vector<Handle*>& handles) {
       uint32_t hash = GetHash(handle);
       LRUCacheShard* shard = static_cast<LRUCacheShard*>(GetShard(Shard(hash)));
       shard->Promote(lru_handle);
+      // Since wait is needed, we assume these handles are still in sec cache.
+      lru_handle->SetIsInSecondaryCache(true);
```

Comment:
If SetIsInSecondaryCache() is always called in LRUCacheShard::Lookup(), then this becomes unnecessary.